internal/deviations/deviations.go

```diff
@@ -625,3 +625,9 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// EcnSameMinMaxThresholdUnsupported returns true for devices that don't support Minimum and Maximum threshold values are same
```

Comment:
... that don't support the same value for Minimum and Maximum threshold.

Reply:
modified